central/detection/service/service_impl.go

```diff
@@ -237,6 +238,12 @@ func (s *serviceImpl) runDeployTimeDetect(ctx context.Context, enrichmentContext
 	if err != nil {
 		return nil, errox.InvalidArgs.New("could not convert to deployment from resource").CausedBy(err)
 	}
+
+	// Deployment ID is empty because the processed yaml comes from roxctl and therefore doesn't
+	// get a  Kubernetes generated ID. This is a temporary ID only required for roxctl to distinguish
```

Comment:
accidental extra space
```suggestion
	// get a Kubernetes generated ID. This is a temporary ID only required for roxctl to distinguish
```